docs/cai/getting-started/MCP.md

```
@@ -0,0 +1,71 @@
+```
+
+https://github.com/user-attachments/assets/386a1fd3-3469-4f84-9396-2a5236febe1f
+
+## Example: Controlling Chrome with CAI
+
+1) Install node, following the instructions on the [official site](https://nodejs.org/en/download/current)
+
+2) Instal Chrome (Chromium is not compatible with this functionality)
+
+3) Run the following commands:
+	```
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)